lib/docs/filters/padrino/entries.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class Padrino
+    class EntriesFilter < Docs::EntriesFilter
+      def get_name
+        node = at_css('h1')
+        result = node.content.strip
+        result << ' event' if type == 'Events'
+        result << '()' if node['class'].try(:include?, 'function')
+        result
+      end
+
+      def get_type
+        return 'Ruby'
```

Comment:
I mean the output in the terminal when generating the HTML

![generated_html_output](https://cloud.githubusercontent.com/assets/264708/15800991/35fd5546-2a88-11e6-89ef-9187b4d5310a.png)


The navigation in the HTML is still missing (think some JS is missing there)

![generated_html](https://cloud.githubusercontent.com/assets/264708/15800994/54d5e65e-2a88-11e6-9a40-9aa0f8ae3272.png)